ape_infura/provider.py

```diff
@@ -92,6 +106,17 @@ def connect(self):
     def disconnect(self):
         self._web3 = None
 
+    def reconnect(self):
+        """
+        Disconnect the connected API.
+        Refresh the API keys from environment variable.
+        Make the self.network_uris empty otherwise the old network_uri will be returned.
+        Connect again.
```

Comment:
Does it connect again though? Is this part of the doc (and the method name) misleading?

Idea:

Delete this method and move

```
    self.load_api_keys()
    self.network_uris = {}
```

to the disconnect method.
thoughts?

Reply:
If we are not connecting then we can move it into `disconnect` method?

Reply:
But I think we need these 2 lines

```py
        self.load_api_keys()
        self.network_uris = {}

```

maybe move it inside `disconnect` ?

Reply:
yes that is my suggestion!